shap/explainers/_tree.py

```diff
@@ -68,53 +73,68 @@ def __init__(
         feature_perturbation="interventional",
         feature_names=None,
         approximate=False,
-        **deprecated_options,
+        # FIXME: Do not allow unexpected arguments.
+        # `link` and `linearize_link` may be passed in by Explainer, but are unused.
+        **kwargs,
     ):
         """ Build a new Tree explainer for the passed model.
 
         Parameters
         ----------
         model : model object
-            The tree based machine learning model that we want to explain. XGBoost, LightGBM, CatBoost, Pyspark
-            and most tree-based scikit-learn models are supported.
+            The tree based machine learning model that we want to explain.
```

Comment:
Did you somehow limit the comment width programmatically, e.g. through a formatting tool?

Reply:
Yes, I used the [rewrap](https://[marketplace.visualstudio.com/items?itemName=stkb.rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap)) VS Code extension. I find wrapping can help long paragraphs be readable both as source code and in the rendered HTML.